.github/workflows/new-label.yml

```diff
@@ -46,6 +46,21 @@ jobs:
           If you require immediate assistance, contact Salesforce Customer Support.
     - name: log action output
       run: echo ${{steps.run_action.outputs.message}}
+  validate-new-issue:
```

Comment:
if I get your auto-response [say I forgot to put my version stuf], and then I modify the issue to include that, will this rerun?  I'm thinking **no** because it's triggered off of new labels.  

We'd probably want it to "maintain" the `more information required` label until the issue meets the criteria (it might not need to comment again if the previous comment still stands, but it could make sure the label stays on)?

Maybe it should be triggered by something else (create/edit of an issue ?)